.github/workflows/review.yml

```diff
@@ -23,13 +23,11 @@ jobs:
           if git diff --name-only origin/${{ github.base_ref }} HEAD | grep -Eq '*\.js|node\.d\.plugin\.in' ; then
```

Comment:
If we’re going to be linting other file types, we need to update this regex to match those files.